frontend/src/components/smart/SkillBalanceDisplay.vue

```diff
@@ -167,20 +433,60 @@ export default Vue.extend({
   },
 
   components: {
-    BModal
+    BModal,
+    ElementTrait
   }
 });
 </script>
 
 <style scoped>
+@media (max-width: 576px) {
+  .custom-skill-balance-mobile{
+    font-size: 3.3vw !important;
+    border-left: 1px solid #424A59;
+  font-size: clamp(.8rem, .7vw, 1rem) !important;
+  }
+  .none-mobile {
+    display: none !important;
+  }
+}
+.claimable {
+  text-align: center;
+  animation: claimable 1s ease-in-out infinite alternate;
```

Comment:
This looks fine on Google Chrome but on Firefox this could cause epilepsy. Either slow it down or remove it altogether please